src/python/dxpy/bindings/dxfile.py

```diff
@@ -38,7 +38,7 @@
 
 DXFILE_HTTP_THREADS = min(cpu_count(), 8)
 MIN_BUFFER_SIZE = 1024*1024
-DEFAULT_BUFFER_SIZE = 1024*1024*16
+DEFAULT_BUFFER_SIZE = 1024*1024*4
```

Comment:
Once you've made the changes suggested below, please change this back to 16MB so as to leave the existing behavior unchanged for AWS users.